scripts/travis-ci/script.bash

```diff
@@ -6,10 +6,18 @@
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
 if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
-	if [ "${MUMBLE_QT}" == "qt4" ]; then
+	if [ "${MUMBLE_QT}" == "qt4" ] && [ "${MUMBLE_HOST}" == "x86_64-linux-gnu" ]; then
 		qmake-qt4 CONFIG+="release tests g15-emulator qt4-legacy-compat" -recursive && make -j2 && make check
-	elif [ "${MUMBLE_QT}" == "qt5" ]; then
+	elif [ "${MUMBLE_QT}" == "qt5" ] && [ "${MUMBLE_HOST}" == "x86_64-linux-gnu" ]; then
 		qmake CONFIG+="release tests g15-emulator" -recursive && make -j2 && make check
+	elif [ "${MUMBLE_QT}" == "qt5" ] && [ "${MUMBLE_HOST}" == "i686-w64-mingw32" ]; then
+		/usr/lib/mxe/usr/bin/${MUMBLE_HOST}.static-qmake -recursive -Wall CONFIG+="release tests g15-emulator"
```

Comment:
Should be:
`/usr/lib/mxe/usr/bin/${MUMBLE_HOST}.static-qmake-qt5 -recursive -Wall CONFIG+="release tests g15-emulator"`
I didn't notice that "qt5" was missing during the previous review.